modules/android/partials/release-notes/couchbase-mobile-android-release-note.3.2.0-beta.adoc

```diff
@@ -0,0 +1,202 @@
+[#maint-3-2-0-beta-1]
+== 3.2.0 Beta 1 -- March 2024
+
+Version 3.2.0 Beta 1 for {param-title} delivers the following features and enhancements:
+
+=== Enhancements
+
+* https://issues.couchbase.com/browse/CBL-4378[CBL-4378 - Add consumer-rules.pro to Android maven and zip distributions]
```

Comment:
This change is mentioned for the main 3.2.0 relnotes too, isn't it?

As both are included, that means that the bug will appear twice on the page?
This seems a bit odd to me.

Reply:
Good spot, will remove it from the GA notes if it's already present